pkg/kubectl/testing/types.generated.go

```diff
@@ -158,7 +158,13 @@ func (x *TestStruct) CodecEncodeSelf(e *codec1978.Encoder) {
 				z.EncSendContainerState(codecSelfer_containerArrayElem1234)
 				if yyq2[2] {
 					yy10 := &x.ObjectMeta
-					yy10.CodecEncodeSelf(e)
+					yym11 := z.EncBinary()
```

Comment:
I did not change these. This is what hack/update-all.sh generated. And the pre-commits were passing.